src/index.jsx

```diff
@@ -1,17 +1,70 @@
-import React from 'react';
+import React,{ useEffect, useState } from 'react';
 import { render } from 'react-dom';
+
+// styles
 import './style.css';
 
+// components
+import AnimalDetail from './components/AnimalDetail/AnimalDetail';
+import AnimalList from './components/AnimalList/AnimalList';
+
 const App = () => {
+  
+  const [listOfAnimals, setListOfAnimals] = useState();
+  const [chosenAnimal, setChosenAnimal] = useState();
+  const [listOfZoos, setListOfZoos] = useState()
+
+  // fetching data - zviratka
+  useEffect(() => {
+    fetch('https://lrolecek.github.io/zviratka-api/zvirata.json')
+    .then(response => response.json())
+    .then(data => {
+      setListOfAnimals(data.zvirata);
+      setChosenAnimal(data.zvirata[0]);
+    });
+  }, [])
+
+  // fetching data - zoo
+  useEffect(() => {
+    fetch('https://lrolecek.github.io/zviratka-api/zoo.json')
+    .then(response => response.json())
+    .then(data => {
+      setListOfZoos(data.zoo);
+    });
+  }, [])
```

Comment:
jak by se dělalo kdybych chtěl počkat až se načtou obě dvě api a až pak updatovat stav? :) (takhle běží každé separátně, můžou trvat jiný čas) (takhle to taky funguje, není to špatně)